code-of-conduct-transparency-reports/2019-07-02-coc-transparency-report.md

```diff
@@ -0,0 +1,29 @@
+# The Carpentries Code of Conduct Transparency Report
+## 2019 July 02
+
+## Overview
+
+### Reports
+
+In the last eight months, the Code of Conduct committee has not received an incident report.
+
+### Potential Code of Conduct Breaches  
+There have been no **reported** Code of Conduct breaches in the last eight months. The Executive Council has approved a task force that will develop recommendtions for incidents that fall outside the scope of the Code of Conduct.
+
```

Comment:
Would the following work?

There have been no **reported** Code of Conduct breaches in the last eight months. However, discussions in the community and within the Code of Conduct committee has led the Executive Council to approve a task force ....

A bit uncertain about the word "discussions", but...

Reply:
Thanks @karinlag!